Sources/TuistLoader/Models+ManifestMappers/ProjectOption+ToManifestMapper.swift

```diff
@@ -6,9 +6,9 @@ extension ProjectDescription.Project.Options {
     /// - Parameters:
     /// - manifest: Manifest representation of project options.
     static func from(manifest: TuistGraph.Project.Options) -> Self {
-        options(
+        projectOptions(
```

Comment:
I'm not sure if we want to deprecate old initializers, I had to add an additional one to keep backward compatibility and allow compilation (default values in the functions caused a compilation issue)

I can definitely add a deprecation warning if we decide to adopt this API